### ISSUE TITLE
There is no export for SharedAccessSignature (and others) from 'azure-storage' module used in main module

### ISSUE THREAD

**Issue by peterblazejewicz** [Azure/azure-sdk-for-node#1616]
The main library still expose SharedAccessSignature (and other modules from Credentials section):
https://github.com/Azure/azure-sdk-for-node/blob/master/lib/azure.js#L750-L754
```
exports.SharedAccessSignature = storage.SharedAccessSignature;
exports.SharedKey = storage.SharedKey;
exports.SharedKeyLite = storage.SharedKeyLite;
exports.SharedKeyTable = storage.SharedKeyTable;
exports.SharedKeyLiteTable = storage.SharedKeyLiteTable;
```
while the `azure-storage` do not export them:
https://github.com/Azure/azure-storage-node/commit/678cfb637095fd5872603f6696a16bd5f6bbb9a7#diff-b162c377f980d391c73e3152b83494a3L195
https://github.com/Azure/azure-storage-node/blob/master/lib/azure-storage.js#L213-L229

For this reason several existing samples just fail, e.g.:
https://github.com/Azure/azure-sdk-for-node/blob/master/examples/samples/sassample.js

Should it be used with `azure-storage-legacy`?
https://github.com/Azure/azure-sdk-for-node/blob/b527caff845a0a4a28eaf08039ad6d5443d5ae54/test/services/blob/internal/sharedaccesssignature-tests.js#L26
https://github.com/Azure/azure-sdk-for-node/tree/b527caff845a0a4a28eaf08039ad6d5443d5ae54/lib/services/legacyStorage

Any transition guide or docs?

Thanks!

**Comment by amarzavery** [comment]
@peterblazejewicz - As per Yang Xia, The feature has been deprecated when we released azure-storage 0.1.0 on 2014.06.12. 

You can talk to Yang Xia in the storage node repo where I had filed the issue for you over here: https://github.com/Azure/azure-storage-node/issues/118. 

I am closing the issue in this repo.

**Comment by yaxia** [comment]
The feature has been deprecated when we released azure-storage 0.1.0 on 2014.06.12. 
Please check the change log [here](https://github.com/Azure/azure-storage-node/blob/master/ChangeLog.txt#L202). If you want to use the storage services, please get azure-storage package from npm and use the SAS token to access the resource.
You might want to check the `generateSharedAccessSignature` function in each service client.
[SAS for blob](http://azure.github.io/azure-storage-node/BlobService.html#generateSharedAccessSignature)
[SAS for table](http://azure.github.io/azure-storage-node/TableService.html#generateSharedAccessSignature)
[SAS for file](http://azure.github.io/azure-storage-node/FileService.html#generateSharedAccessSignature)
[SAS for Queue](http://azure.github.io/azure-storage-node/QueueService.html#generateSharedAccessSignature)

**Comment by peterblazejewicz** [comment]
Thanks!
